SimulationConditions/inc/SimBookkeeperMaker.hh

```diff
@@ -0,0 +1,43 @@
+#ifndef SimulationConditions_SimBookkeeperMaker_hh
+#define SimulationConditions_SimBookkeeperMaker_hh
+
+//
+// Makes the SimBookkeeper ProditionsEntitiy
+//
+
+#include "SimulationConditions/inc/SimBookkeeper.hh"
+#include "SimulationConfig/inc/SimBookkeeperConfig.hh"
+#include "DbTables/inc/SimEfficiencies.hh"
+
+namespace mu2e {
+
+  class SimBookkeeperMaker {
+  public:
+    SimBookkeeperMaker(SimBookkeeperConfig const& config):_config(config) {}
+
+    typename SimBookkeeper::ptr_t fromFcl() {
```

Comment:
I don't recognize this - what is typename doing here?

Reply:
This looks like a copy error (I copied from TrkQualCatalogMaker which uses templates). I've removed "typename" from the file